test/convnets.jl

```diff
@@ -6,120 +6,114 @@
     _gc()
 end
 
-@testset "VGG" begin
-    @testset "VGG($sz, batchnorm=$bn)" for sz in [11, 13, 16, 19], bn in [true, false]
-        m = VGG(sz, batchnorm = bn)
-        @test size(m(x_224)) == (1000, 1)
-        if (VGG, sz, bn) in PRETRAINED_MODELS
-            @test acctest(VGG(sz, batchnorm = bn, pretrain = true))
-        else
-            @test_throws ArgumentError VGG(sz, batchnorm = bn, pretrain = true)
-        end
-        @test gradtest(m, x_224)
-        _gc()
+@testset "VGG" begin @testset "VGG($sz, batchnorm=$bn)" for sz in [11, 13, 16, 19],
+                                                            bn in [true, false]
+
+    m = VGG(sz; batchnorm = bn)
+    @test size(m(x_224)) == (1000, 1)
+    if (VGG, sz, bn) in PRETRAINED_MODELS
+        @test acctest(VGG(sz; batchnorm = bn, pretrain = true))
+    else
+        @test_throws ArgumentError VGG(sz, batchnorm = bn, pretrain = true)
     end
-end
+    @test gradtest(m, x_224)
+    _gc()
+end end
```

Comment:
I think you ran the formatter on the full test suite again. Can you only run it on `src` like @theabhirath suggested?